source/_integrations/nuki.markdown

```diff
@@ -24,9 +24,20 @@ The Nuki integration allows you to control [Nuki Smart Locks](https://nuki.io/en
 ## Prerequisites
 
 To add a Nuki bridge to your installation, you need to enable developer mode on your bridge and define a port and an access token. This can be achieved using the [Android app](https://play.google.com/store/apps/details?id=io.nuki) or [iPhone app](https://apps.apple.com/app/nuki-smart-lock/id1044998081). Go to manage my devices, and select the bridge. Within the bridge configuration turn on the HTTP API and check the details in the screen. Please note that the API token should be 6-20 characters long, even though the app allows you to set a longer one.
+For faster updates, the callback function of the Nuki bridge can be used. This requires your Home Assistant to be reachable via HTTP by the Nuki bridge, as HTTPS is not supported by the Nuki bridge.
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+  Host:
+    description: Hostname or IP address of your Nuki bridge, e.g., `192.168.1.25`.
+  Port:
+    description: Port of the Nuki bridge HTTP API.
+    default: 8080
```

Comment:
`configuration_basic` has no additional modifiers besides `description`. Incorporate the default port into the description instead.

Reply:
Changed it, got it from https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/balboa.markdown?plain=1#L46 (so it probably needs changing there too)